### PR TITLE
Fix page management

### PR DESCRIPTION
**Pull request by fkroockmann**
https://github.com/backbee/backbee-standard/issues/421

### CODE REVIEW

File: src/tb/i18n/fr_FR/global.json [hunk 1]
```diff
@@ -118,6 +118,7 @@
     "scheduling": "Planification",
     "search": "Rechercher",
     "search_content": "Rechercher un contenu",
+    "see": "Voir",
```
Comment (mickaelandrieu) [inline src/tb/i18n/fr_FR/global.json]: :+1: thanks

File: src/tb/apps/page/views/page.view.manage.js [hunk 2]
```diff
@@ -202,27 +220,32 @@ define([
         },
 
         bindGroupedActions: function () {
-            var selectAll = jQuery('#bb-page-manage-select-all'),
-                select = jQuery('#bb-page-manage-grouped-action');
+            var self = this;
 
-            // Don't work correctly
```
Comment (mickaelandrieu) [inline src/tb/apps/page/views/page.view.manage.js]: LOL !

**Comment by mickaelandrieu** [comment]
:+1:

File: src/tb/apps/page/views/page.view.manage.js (hunk 2 above)
Comment (fkroockmann) [inline src/tb/apps/page/views/page.view.manage.js]: x)

**Comment by fkroockmann** [comment]
thanks for review @mickaelandrieu :)

**Pull request merged**